desc/compute/utils.py

```diff
@@ -315,7 +315,10 @@ def get_params(keys, obj, has_axis=False, **kwargs):
     params = []
     for key in deps:
         params += data_index[p][key]["dependencies"]["params"]
-    params = _sort_args(list(set(params)))
+    if p == "desc.equilibrium.equilibrium.Equilibrium":
+        # probably need some way to distinguish between params from different instances
+        # of the same class?
+        params = _sort_args(list(set(params)))
```

Comment:
Why only sort the args if it is an Equilibrium?

Reply:
Right now the equilibrium is the only thing we can optimize, so its the only thing that needs a canonical ordering. This is generalized for other objects in #604